util/env_posix.cc

```diff
@@ -675,6 +675,10 @@ class PosixEnv : public Env {
 
   virtual void SleepForMicroseconds(int micros) override { usleep(micros); }
 
+  virtual void SleepForNanoseconds(int nanos) override {
```

Comment:
If we need to add a function in Env anyway, maybe adding Env::NowMicrosMonotonic() is better than sleeping for nanoseconds.